src/comm/aca_async_grpc_server.cpp

```diff
@@ -1,126 +0,0 @@
-/*
- *
- * Copyright 2015 gRPC authors.
- * Copyright 2019 The Alcor Authors - file modified.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- */
-
-#include <memory>
-#include <iostream>
-#include <string>
-#include <thread>
-
-#include <grpcpp/grpcpp.h>
-#include <grpc/support/log.h>
-#include "goalstateprovisioner.grpc.pb.h"
-#include "aca_comm_mgr.h"
-#include "aca_log.h"
-#include "aca_async_grpc_server.h"
-
-static char GRPC_SERVER_ADDRESS[] = "0.0.0.0:50001";
-
-using namespace alcor::schema;
-using aca_comm_manager::Aca_Comm_Manager;
-using grpc::Server;
-using grpc::ServerAsyncResponseWriter;
-using grpc::ServerBuilder;
-using grpc::ServerCompletionQueue;
-using grpc::ServerContext;
-using grpc::Status;
-
-static std::atomic<bool> running;
-Aca_Async_GRPC_Server::~Aca_Async_GRPC_Server()
```

Comment:
is this a legacy implementation?

Reply:
yes, that's the old implementation, not being compiled anymore.